File: Ejercicios/Ejercicio 31/JS/31.js
let numero;
let suma=0;
let mensaje;
numero= prompt();

for (i=1;i<numero;i++){
    if (numero % i == 0){
        suma=suma+1;
    }
}
if (suma==1){
    mensaje=" Es un numero primo";
} else{
    mensaje=" No es un numero primo";
}
document.write(numero,mensaje);

File: Ejercicios/Ejercicio 31/C/31.c
#include <stdio.h>

int main()
    {
    int i,numero,suma=0;

    printf("Ingresa un numero para determinar si es primo, o no. \n");
    scanf("%d",&numero);
    for (i=1;i<numero;i++) {
        if (numero % i == 0) {
            suma=suma+1;
        }
    }
    if (suma==1) {
        printf("%i Es un numero primo",numero);
    } else {
        printf("%i No es un numero primo",numero);
    }

    return 0;
    }
